iceoryx_examples/waitset_in_c/ice_c_waitset_timer_driven_execution.c

```diff
@@ -120,7 +117,7 @@ int main()
     // start a thread which triggers cyclicTrigger every second
     //! [cyclic trigger thread]
     pthread_t cyclicTriggerThread;
-    if (createThread(&cyclicTriggerThread, cyclicTriggerCallback))
+    if (!createThread(&cyclicTriggerThread, cyclicTriggerCallback))
```

Comment:
Can you please update the documentation and remove the note which tells that this does not run on Windows?